README.md

```diff
@@ -344,8 +344,35 @@ For a comparison of some of these, see this
 
 ## Storage Backends
 
-In order to not download or scan any previously scanned sources again, the _scanner_ can use a storage backend to store
-scan results for later reuse.
+In order to not download or scan any previously scanned sources again or to reuse scan results generated via other
+services, the _scanner_ can be configured to use so-called storage backends. Before processing a package, it checks
+whether compatible scan results are already available in one of the storages declared; if this is the case, they
+are downloaded and reused. Otherwise, the package is scanned now. Afterwards, the new results can be put into a
```

Comment:
Nit: I'd drop "now".

Reply:
Also, maybe more explicit "Otherwise, the package's source code is downloaded and scanned."

"Afterwards, the new scan results..." (insert "scan")